### DISCUSSION TITLE
How i can do a question

### DISCUSSION THREAD

**Discussion by Windows1Door2Daniel** [FunkinCrew/Funkin#1543] [Q&A]
how i do it

u got trolled

 why week 2 is so hard in normal dificulty
https://www.youtube.com/watch?v=dQw4w9WgXcQ

**Comment by ghost** [comment]
The Adidas ad saved me

**Comment by Windows1Door2Daniel** [comment]
> The Adidas ad saved me

dang my most powerful weapon

**Comment by ghost** [comment]
Keep on forgetting to get uBlock Origin. This forgetfulness has saved me so many rickrolls

**Comment by Windows1Door2Daniel** [comment]
mmh

**Comment by Windows1Door2Daniel** [comment]
i didnt even get responded why week 2 is so hard af

**Comment by ghost** [comment]
I can do 2 on hard

It;s second hardest week but yeah its hard
